.nvmrc

```diff
@@ -1 +1 @@
-16.14.0
\ No newline at end of file
+18
```

Comment:
we can separate this but I think it's a good occasion to pull up the node version, too.